### PR TITLE
Modify readme

### PR DESCRIPTION
**Pull request by Mossaka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,13 +142,17 @@ SpiderLightning's interfaces offer developers a set of provider-agnostic APIs wh
...
+
+SpiderLightning has evolved to `wasi-cloud-core` and is now being standardized under the WASI Subgroup. See [here](https://github.com/WebAssembly/wasi-cloud-core)
 
 ### What is the difference between SpiderLightning and Dapr?
 
-SpiderLightning and Dapr share the same goal of providing capabilities to distributed applications, but each project's approach to achieving this goal is very different. For example, while Dapr runs as a sidecar container and provides an HTTP/gRPC interface for applications, SpiderLightning chooses not take a dependency on network transport protocols. Instead, `slight` links applications directly to generated WASM bindings, then executes them directly.
+SpiderLightning and Dapr share the same goal of providing capabilities to distributed applications, but each project's approach to achieving this goal is very different. For example, while Dapr runs as either a sidecar container or one-per-node container and provides an HTTP/gRPC interface for applications, `slight` links applications directly to generated WASM bindings, which means calls executed in-process do not depend on a particular network transport protocol.
```
Comment (squillace) [inline README.md:155]: just `which means all calls are executed in-process.` 

that's it. doubt we need the re-assertion that we're removing a network call by doing so.

**Review by squillace — CHANGES_REQUESTED** [review]
fix that one line and ship it!

**Review by danbugs — APPROVED** [review]
I think this looks good to merge after fixing the one typo Ralph pointed out 👍

File: README.md (hunk 1 above)
Comment (Mossaka) [reply README.md]: Done

**Pull request merged**